software_center.rst

```diff
@@ -1,84 +1,144 @@
 .. _software_center-section:
 
 ===============
-Software center
+Software Center
 ===============
 
-The software center lists applications available from all repositories
-[#fileserver]_\ .
+The Software Center displays and manages applications available from all
+configured repositories [#fileserver]_\ .
 
-To install a new application just click on the :guilabel:`Install` button.
-If the cluster has multiple nodes, you will be alse required to select the
-target node.
+Application information
+========================
 
-To install another instance of an existing application, click on the
-``Instances`` link inside the application card. Then click on
-:guilabel:`Install new instance`.
+Each application is represented by a card displaying its name, level of
+certification, and category. Click on the application's name to view
+detailed information such as screenshots, author, latest version,
+container images, and links to documentation, source code, and bug
```

Comment:
I'd delete the first " and":
container images,  links to documentation, source code, and bug